vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ImageService.java

```diff
@@ -126,23 +124,23 @@ public List<IVSImage> getImages(int pageNo) {
      */
 
     @Override
-    public List<IVSImage> getImages(int pageNo, ImageCategory category, String sortedBy, String order) {
+    public List <IVSImage> getImages(int pageNo, ImageCategory category, String sortedBy, String order) {
         Sort sortingParameters = getSortingParameters(sortedBy, order);
         pageNo = validatePageNumber(pageNo, category);
         Pageable sortByRequestedField = PageRequest.of(pageNo - 1, pageSize, sortingParameters);
-        Page<VSImage> images;
-        if(category==null) {
+        Page <VSImage> images;
+        if (category == null) {
             images = imageRepo.findAll(sortByRequestedField);
         } else {
             images = imageRepo.findByCategories(sortByRequestedField, category);
         }
-        List<IVSImage> results = new ArrayList<>();
-        if(images != null) {
+        List <IVSImage> results = new ArrayList < > ();
```

Comment:
there are multiple instance of blanks before `<...>` in here

Reply:
i have removed them. In ImageService.java there were bunch of spaces.